boards/xtensa/qemu_xtensa/qemu_xtensa_defconfig

```diff
@@ -10,3 +10,4 @@ CONFIG_SW_ISR_TABLE=y
 CONFIG_GEN_ISR_TABLES=y
 CONFIG_GEN_IRQ_VECTOR_TABLE=n
 CONFIG_SIMULATOR_XTENSA=y
+CONFIG_LOG=n
```

Comment:
we should not be disabling features like this on the board level. Why don't we fix this for xtensa instead? There is a bug open about this actually: #10164